src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java

```diff
@@ -39,7 +39,7 @@ public class T_104CoMoOutboundCallDetails {
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long iD;
 	private Integer uSRMappingID;
-	private String czentrixCallID;
+	private String callCentreID;
```

Comment:
@Vidyaaa24 Please change this property to callId. It is not callCentreId. if you look at the original property, it was czentrixCallId.